pennylane/ops/op_math/controlled.py

```diff
@@ -67,8 +71,8 @@ def ctrl(op, control: Any, control_values=None, work_wires=None):
     Args:
         op (function or :class:`~.operation.Operator`): A single operator or a function that applies pennylane operators.
         control (Wires): The control wire(s).
-        control_values (bool or list[bool]): The value(s) the control wire(s) should take.
-            Integers other than 0 or 1 will be treated as ``int(bool(x))``.
+        control_values (bool or list[bool] or int or list[int]): The value(s) the control wire(s)
```

Comment:
This is super nitpicky at this point, but I guess we allow mixed lists as well, also indicated by the type hint?
```suggestion
        control_values (bool or int or list[bool or int]): The value(s) the control wire(s)
```